package.json

```diff
@@ -96,6 +100,7 @@
     "rimraf": "^3.0.2",
     "storybook": "^6.2.8",
     "storybook-design-token": "^1.0.1",
+    "style-dictionary": "3",
```

Comment:
This version notation is different from our project standard. Was this intentional?

Reply:
At the time I installed it 2.x was the latest stable version, so I had run `yarn add style-dictionary@3` to get the latest version. Semantically `3` is identical to `3.0.0`, so I don't think it is an issue per se, and it will probably be replaced by a newer version by Dependabot sometime soon anyway.